lib/PCAP/Threaded.pm

```diff
@@ -235,9 +235,11 @@ sub external_process_handler {
       system("/usr/bin/time bash $script 1> $out 2> $err");
     }
     catch {
-      warn "\nGeneral output can be found in this file: $out\n";
-      warn "Errors can be found in this file: $err\n\n";
-      die "Wrapper script message:\n".$_;
+      warn "\nTHREAD_EXITED: General output can be found in this file, last 10 lines below: $out\n";
```

Comment:
As discussed wit @drjsanger on Slack, makes viewing of logs in wrapping schedulers etc easier.